subiquity/server/controller.py

```diff
@@ -54,6 +57,26 @@ async def _confirmed(self):
             await self.configured()
             self._active = False
 
+    def validate_autoinstall(self, ai_data: dict) -> None:
+        try:
+            jsonschema.validate(ai_data, self.autoinstall_schema)
+
+        except ValidationError as original_exception:
+            section = self.autoinstall_key
+
+            new_exception: AutoinstallValidationError = AutoinstallValidationError(
+                section,
+                original_exception,
+            )
+
+            self.app.make_apport_report(
```

Comment:
By default, autoinstall errors should not be apport reports.  This sort of flaw should move to manual reporting - we have this problem today were invalid data can trigger the bug flow and we don't want that.  Most invalid autoinstall is a user configuration error.  Select "invalid" autoinstall is really a bug, so we want to rely on manual user reporting for that case.

Reply:
This is true. I'll flip this around and add some special handling in the `SubiquityServer` exception handler to not write out apport reports on instances of `AutoinstallError`.